pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -6985,6 +6985,18 @@ final: prev:
     meta.homepage = "https://github.com/lawrence-laz/neotest-zig/";
   };
 
+  neotree-nvim = buildNeovimPlugin {
+    pname = "neotree.nvim";
+    version = "2024-05-22";
+    src = fetchFromGitHub {
+      owner = "nvim-neotree";
+      repo = "neotree.nvim";
+      rev = "29f7c215332ba95e470811c380ddbce2cebe2af4";
+      sha256 = "1k3fhgpakr1b8x9fk7sq699h3479k8x9yc3jbq7xq2jk7p0n4m62";
+    };
+    meta.homepage = "https://github.com/nvim-neo-tree/neo-tree.nvim/";
+  };
+
```

Comment:
Does this have anything to do with the [neo-tree plugin](https://github.com/NixOS/nixpkgs/blob/543582d5e2c38f1577f59c6e75e9f97d748b1c07/pkgs/applications/editors/vim/plugins/generated.nix#L6482-L6492) we already have?

Reply:
You're right, I missed that on `search.nixos.org`